app/App.hs

```diff
@@ -293,23 +282,29 @@ runPipelineOptions m = do
           }
   runReader opt m
 
-runPipelineEntry :: (Members '[App, ProgressLog, EmbedIO, TaggedLock] r) => EntryPoint -> Sem (PipelineEff r) a -> Sem r a
+runPipelineEntry :: (Members '[App, Logger, EmbedIO, TaggedLock] r) => EntryPoint -> Sem (PipelineEff r) a -> Sem r a
 runPipelineEntry entry p = runPipelineOptions $ do
   r <- runIOEither entry (inject p) >>= fromRightJuvixError
   return (snd r ^. pipelineResult)
 
 runPipelineSetup ::
-  (Members '[App, EmbedIO, Reader PipelineOptions, TaggedLock] r) =>
+  (Members '[App, EmbedIO, Logger, Reader PipelineOptions, TaggedLock] r) =>
   Sem (PipelineEff' r) a ->
   Sem r a
-runPipelineSetup p = ignoreProgressLog $ do
+runPipelineSetup p = do
   args <- askArgs
   entry <- getEntryPointStdin' args
   r <- runIOEitherPipeline entry (inject p) >>= fromRightJuvixError
   return (snd r)
 
+-- say :: (Member App r) => Text -> Sem r ()
```

Comment:
Can be removed? (I can't delete using GitHub suggestion because there's no way to remove the extra newline)

Reply:
I forgot to remove that. It is now fixed